list.c

```diff
@@ -192,7 +201,9 @@ void vListInsert( List_t * const pxList,
         for( pxIterator = ( ListItem_t * ) &( pxList->xListEnd ); pxIterator->pxNext->xItemValue <= xValueOfInsertion; pxIterator = pxIterator->pxNext )
         {
             /* There is nothing to do here, just iterating to the wanted
-             * insertion position. */
+             * insertion position. 
+             * IF YOU FIND YOUR CODE STUCK HERE, SEE THE NOTE JUST ABOVE.
+             */
```

Comment:
Let's remove this. It's 10 lines from the comment block above explaining common causes.

Reply:
Here's the view from the MPLAB X IDE while debugging and halting on this for loop, on which the code is stuck. My eyes naturally focus on the contents of the for loop, in yellow: 

![image](https://github.com/FreeRTOS/FreeRTOS-Kernel/assets/6842199/4125d8ab-63eb-42fe-8f2d-3128fb573b1b)

The purpose of this line:

> IF YOU FIND YOUR CODE STUCK HERE, SEE THE NOTE JUST ABOVE.

Is to tell me to look at the notes above, or else I won't, because I'm looking inside the for loop in which I'm stuck, instead.